modules/apps/27-interchain-accounts/types/expected_keepers.go

```diff
@@ -36,5 +36,5 @@ type PortKeeper interface {
 
 // ParamSubspace defines the expected Subspace interface for module parameters.
 type ParamSubspace interface {
-	GetParamSet(ctx sdk.Context, ps paramtypes.ParamSet)
```

Comment:
Sorry to be a pain but can we keep `GetParamSet` since it would technically be an API breaking change? Happy to just add the `IfExists` method as addition, but I think we should keep this in order to preserve our release guarantees when backporting to v8.3